enterprise/app/executors/executors.css

```diff
@@ -48,6 +48,22 @@
   margin-top: 8px;
 }
 
+.pool-view-executions-button {
+  overflow: hidden;
+}
+
+.pool-view-executions-text {
```

Comment:
let's just leave this visible - we have the same button down below and it's default-shown, and we have plenty of real-estate here.